src/utils/quick-open.js

```diff
@@ -70,7 +71,7 @@ export function formatSourcesForList(source: RelativeSource, tabs: TabList) {
 export type QuickOpenResult = {|
   id: string,
   value: string,
-  title: string,
+  title: string | React$Element<"div">,
```

Comment:
when we use fuzzaldrin to fuzzy highlight we return a React Element

Reply:
👍